src/components/SearchModal/CommonBases.tsx

```diff
@@ -17,9 +18,17 @@ const MobileWrapper = styled(AutoColumn)`
   `};
 `
 
-const BaseWrapper = styled.div<{ disable?: boolean }>`
-  border: 1px solid ${({ theme, disable }) => (disable ? 'transparent' : theme.deprecated_bg3)};
-  border-radius: 10px;
+const BaseWrapper = styled.div<{ disable?: boolean; phase0Flag?: boolean }>`
+  border: 1px solid
+    ${({ theme, disable, phase0Flag }) =>
+      disable
+        ? phase0Flag
+          ? theme.accentAction
+          : 'transparent'
```

Comment:
this is nit-picky but there's a 'theme.none' if u wanna use that lmao

Reply:
oooh ty done